### PR TITLE
EN-2309	fixing scroll using mouse wheel on sidebar

### PR DESCRIPTION
**Pull request by maitojepoy**
included flickering issue when scrolling down, added test to omit getboundingcoords

### CODE REVIEW

**Comment by entando-service** [comment]
Can one of the admins verify this patch?

File: src/ui/pages/config/ToolbarPageConfig.js [hunk 1]
```diff
@@ -51,7 +56,19 @@ class ToolbarPageConfig extends Component {
           </span>
         </span>
         <div className="panel-group">
-          <div className="ToolbarPageConfig__drawer-pf-container" >
+          <div
+            className={classScrollContainer.join(' ')}
+            onMouseEnter={(
+              () => {
+                document.body.style.overflow = 'hidden';
```
Comment (gidesan) [inline src/ui/pages/config/ToolbarPageConfig.js]: please use classes instead of setting DOM properties though JS

Comment (jeffgo10) [reply src/ui/pages/config/ToolbarPageConfig.js]: done. please review @gidesan

**Comment by pietrangelo** [comment]
retest this please

**Pull request merged**